src/components/ConfirmationModal.tsx

```diff
@@ -27,7 +27,7 @@ import { type ButtonVariant } from "react-bootstrap/types";
 
 type ModalProps = {
   title?: string;
-  message: string;
+  message: string | React.ReactNode;
```

Comment:
```suggestion
  message: React.ReactNode;
```
`string` is valid for `ReactNode`. It also allows `boolean`, `number`, and several other types.

A more narrow type would be:
```suggestion
  message: string | React.ReactElement;
```

Reply:
TIL the difference between the two, thank you for pointing this out